packages/ndk-sysroot/math-header.diff

```diff
@@ -0,0 +1,22 @@
+--- data/data/com.termux/files/usr/include/c++/v1/cmath.orig	2019-03-24 00:31:49.512145195 +0000
++++ ./data/data/com.termux/files/usr/include/c++/v1/cmath	2019-03-24 00:32:19.392703442 +0000
+@@ -302,7 +302,7 @@
+ */
+ 
+ #include <__config>
+-#include <math.h>
++#include <c++/v1/math.h>
```

Comment:
Why not `#include "math.h"`?

Reply:
Because it was done this way last time....